lib/iris/tests/__init__.py

```diff
@@ -59,8 +59,8 @@
 try:
     import matplotlib
 
-    matplotlib.use("agg")
     matplotlib.rcdefaults()
+    matplotlib.use("agg")
```

Comment:
@rcomer Awesome spot :eyes: 

Works for me :+1: 

Would you mind just adding a comment above `matplotlib.use("agg")` to ensure that a future developer doesn't move this back to being before `matplotlib.rcdefaults()`, thanks :grinning:

Reply:
Otherwise LGTM :partying_face:

Reply:
closes #3821

Reply:
Thanks @bjlittle, I've added that comment.  Also a comment about why the `rcdefaults` is there for good measure.